debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -136,7 +136,7 @@ public void shouldReceiveChangesForInsertsWithDifferentDataTypes() throws Except
 
         //date and time
         consumer.expects(1);
-        assertInsert(INSERT_DATE_TIME_TYPES_STMT, 1, schemaAndValuesForDateTimeTypes());
+        assertInsert(INSERT_DATE_TIME_TYPES_STMT, 1, schemaAndValuesForDateTimeTypesAdaptiveTimeMicroseconds());
```

Comment:
Why has this changed? Does it indicate a change to the default behavior?

Reply:
IIRC we've decided to drop adaptive mode. This was removed in MySQL but was kept in Postgres so this is unification for all three connectors and this has impact on postgres default

Reply:
Hum, I can't remember that we decided this. Are you sure?

Reply:
@gunnarmorling https://github.com/debezium/debezium/pull/864/files#diff-d565df219003c24920bee4de756b0324R1172